kuma/javascript/src/payments/strings.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { gettext, interpolate } from '../l10n.js';
+
+export const strings = {
+    signup: 'Become a monthly supporter',
+    signupDesc:
+        'Support MDN with a $5 monthly subscription and get back more of the knowledge and tools you rely on for when your work has to work.',
```

Comment:
To future-proof ourselves, can you move that amount out and use `interpolate()` here even if we don't do any fancy formatting of the number `5`. E.g. 
```javascript
interpolate(gettext(`Support MDN with a <amount/> monthly subscription ...`, {amount: '$5'})
```

Reply:
Changed and added a link to the GH issue 👍